spi/data-plane/data-plane-spi/src/main/java/org/eclipse/edc/connector/dataplane/spi/pipeline/DataSinkFactory.java

```diff
@@ -35,12 +35,19 @@ public interface DataSinkFactory {
 
     /**
      * Returns true if the request is valid.
+     *
+     * @deprecated use {@link #validateRequest(DataFlowRequest)} instead.
      */
-    @NotNull Result<Boolean> validate(DataFlowRequest request);
+    @Deprecated
```

Comment:
please add the `since` attribute with "milestone9" value so we'll know how old it is